tests/zfs-tests/tests/functional/cli_root/zfs_receive/zfs_receive_raw_incremental.ksh

```diff
@@ -49,6 +49,7 @@ function cleanup
 		log_must zfs destroy -r $TESTPOOL/$TESTFS2
 
 	[[ -f $ibackup ]] && log_must rm -f $ibackup
+	[[ -f $ibackup_trunc ]] && log_must rm -f $ibackup_trunc
```

Comment:
We should also change ibackup and ibackup_trunc's location to $TEST_BASE_DIR.  Which is normally `/var/tmp` but it set by the buildbot to an alternate path for its testing.

Reply:
Done thanks!